src/verifyEmail.js

```diff
@@ -0,0 +1,27 @@
+function verifyEmail(appId, serverURL) {
```

Comment:
Yeah some parts of those are potential customization points. I think it might make more sense as a separate adapter though.

If a class is never overridden, doesn't inherit any meaningful behaviour, and doesn't have any instance-specific data, I don't see any need to put it in a class. Classes are also slower due to chasing pointers around the prototype chain, if you have few instantiations and many calls (this applies to our controllers)